src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -378,14 +378,14 @@ class VAMap extends Component {
     const { results } = this.props;
 
     // need to use this because Icons are rendered outside of Router context (Leaflet manipulates the DOM directly)
-    const linkAction = (id, isProvider = false, e) => {
-      e.preventDefault();
-      if (isProvider) {
-        this.context.router.push(`provider/${id}`);
-      } else {
-        this.context.router.push(`facility/${id}`);
-      }
-    };
+    // const linkAction = (id, isProvider = false, e) => {
```

Comment:
Best I can tell, this function is only being used to update the URL when users click a popup link.

Reply:
Hm, I'm seeing a runtime error on my localhost when I click the link in the popup. I think this is what the comment is referring to -

![image](https://user-images.githubusercontent.com/1915775/60036264-d0850380-967c-11e9-9719-60d2b315d584.png)

Reply:
I think we should leave `linkAction`, but revise it to be -

```jsx
const linkAction = (event) => {
    event.preventDefault();
    this.context.router.push(event.target.href);
};
```